google_bigquery/plugin.spec.yaml

```
@@ -0,0 +1,96 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: google_bigquery
+title: Google BigQuery
+description: Uses the Google BigQuery plugin to run queries on google cloud platform
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/google_bigquery
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+  vendor_url: https://www.google.com/
+tags:
+- google
+- bigquery
+hub_tags:
+  use_cases: [query]
+  keywords: [google, bigquery]
+  features: []
+connection:
+  project_id:
+    title: Project ID
+    type: string
+    description: Project ID from service credentials
+    required: true
+    example: spherical-voice-171717
+  private_key_id:
+    title: Private Key ID
+    type: string
+    description: Private Key ID from service credentials
+    required: true
+    example: 18181818e18181c181d1e18cee1b8e18c1818d1a
+  private_key:
+    title: Private Key
+    type: credential_asymmetric_key
+    description: Private Key from service credentials
+    required: true
+    example: "{\"privateKey\": \"-----BEGIN PRIVATE KEY-----\\nMIIEvQIBEFENByuihkiY9w0BAQAZAAAAAAbb3AbHDbS09uUlXOLPH\\n+AAAAAAA1bbAAAAAbAbb11=\\n-----END PRIVATE KEY-----\\n}"
+  client_email:
+    title: Client Email
+    type: string
+    description: Client email from service credentials
+    required: true
+    example: "user@example.com"
+  client_id:
+    title: Client ID
+    type: string
+    description: Client ID
+    required: true
+    example: "111111111111111111111"
+  auth_uri:
+    title: Auth URI
+    type: string
+    description: OAUTH2 Auth URI
+    default: https://accounts.google.com/o/oauth2/auth
+    required: true
+    example: "https://accounts.google.com/o/oauth2/auth"
+  auth_provider_x509_cert_url:
+    title: Auth Provider X509 Cert URL
+    type: string
+    description: OAUTH2 Auth Provider x509 Cert URL
+    default: https://www.googleapis.com/oauth2/v1/certs
+    required: false
+    example: https://www.googleapis.com/oauth2/v1/certs
+  client_x509_cert_url:
+    title: Client Cert URL
+    type: string
+    description: Client certificate URL from service credentials
+    required: true
+    example: "https://www.googleapis.com/robot/v1/metadata/x509/user%40example.com"
+  token_uri:
+    title: Token URI
+    type: string
+    description: OAUTH2 Token URI
+    default: https://oauth2.googleapis.com/token
+    required: false
+    example: https://oauth2.googleapis.com/token
+actions:
+  query:
+    title: Query
+    description: Get all results from query
+    input:
+      query:
+        title: Query
+        description: Domain to retrieve users from
+        type: string
+        required: true
+        example: "SELECT * FROM (SELECT \"apple\" AS fruit, \"carrot\" AS vegetable)"
```

Will this work with single-quotes instead of the escaped double-quotes? so it matches the UI input where you don't have to escape

```suggestion
        example: "SELECT * FROM (SELECT 'apple' AS fruit, 'carrot' AS vegetable)"
```